src/components/table/TableView.svelte

```diff
@@ -47,6 +47,12 @@
     'labeled_counter',
   ]);
 
+  const PERCENTILES = [5, 25, 50, 75, 95, 99, 99.9];
+
+  visibleBuckets = METRICS_WITH_PERCENTILE_DATA.has(data[0].metric_type)
+    ? PERCENTILES
+    : visibleBuckets;
```

Comment:
IMO it's not ideal to redefine input into a component.

I'm feeling like it makes more sense to do this in the higher level, containing component which already has a fair amount of business logic and keep this one simpler: https://github.com/mozilla/glam/blob/main/src/components/table/ProbeTableView.svelte#L69

There's already a definition of percentiles there. Might make sense to move `METRICS_WITH_PERCENTILE_DATA` to `src/constants.js` while you're at it.